src/app/app.component.ts

```diff
@@ -21,6 +21,7 @@ export class OrganizerAppComponent {
 
   @ViewChild(Nav) public nav: Nav;
   public rootPage: any;
+  private disconnectSubscription: any;
```

Comment:
using `import {Subscription} from "rxjs";` 
giving error ` Type 'Subscription' is not assignable to type 'Subscription'. Property '_parent' is missing in type
            'Subscription'.`
what can be the problem?